tests/SessionTest.php

```diff
@@ -83,34 +101,10 @@ public function testRestart()
         $this->session->restart();
     }
 
-    public function testVisitWithoutRunningSession()
```

Comment:
But IMHO it doesn't matter in this case as we are finde if started() is called and afterwards the visit() method is called. Otherwise tests would need to be duplicated for the other autostart methods as well. 

If you want I can add this in.